### PR TITLE
Experiments

### PR DESCRIPTION
**Pull request by codingkaveman**
Started the project over just to make sure I followed the instructions to the letter this time.

### CODE REVIEW

File: advice/steve-m-advice.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                <p>You see when you encounter a new programming language part of your learning process includes learning to train your eye. The way to train your eye is to read a good deal of code even when you don't totally understand what it's doing.</p>
+            </div>
+        </section>
+
+        <footer>
+            <!-- replace Skillcrush with your name here -->
+            <p>© Copyright Steve Manning 2018.</p>
+        </footer>
+    </body>
+</html>
```
Comment (kerolloz) [inline advice/steve-m-advice.html]: please use newlines at end of files

Comment (codingkaveman) [reply advice/steve-m-advice.html]: newlines? I'm not absolutely sure I understand what you mean by that, can you please elaborate more on exactly what that means?

Comment (kerolloz) [reply advice/steve-m-advice.html]: would you please read this question on  [stack overflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Comment (codingkaveman) [reply advice/steve-m-advice.html]: I've read and re-read it a few times and I'm still confused on what you're telling me to do. Is it adding a <br> line-break at the end of my footer paragraph?

**Comment by acodeinprogress** [comment]
Hey @codingkaveman,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

xoxo,
Sharon

**Pull request merged**